src/testpackage/Practice02.java

```diff
@@ -21,11 +22,17 @@ public static void main(String[] args) {
         System.out.println(sum);
         System.out.println();
 
+        //課題2（IntStream使用：1～10までを足して出力　IntStreamを使用して1行におさめる
+        System.out.println("【課題2+】1~10までを加算して出力 IntStreamを使い1行におさめる");
+        int sum2 = IntStream.of(1,2,3,4,5,6,7,8,9,10).sum();
```

Comment:
1から10をベタに書かなくてもこのようにすればよいかと。
試してください！

```suggestion
        int sum2 = IntStream.rangeClsed(1, 10).sum();
```

Reply:
こんなメソッドがあったとは！
Streamのメソッドを色々調べてみたいと思います。
修正版をアップロードしました。